src/Configuration/CliOptions.cs

```diff
@@ -71,7 +71,12 @@ public static (PlcSimulation PlcSimulationInstance, List<string> ExtraArgs) Init
                         throw new OptionException("The max OPC string length must be larger than 0.", "opcmaxstringlen");
                     }
                 }
-            },
+            },
+
+            // OTLP configuration
+            { "otlpe|otlpendpoint=", $"the endpoint Uri to which the otlp exporter is going to send traces .\nDefault: {config.OtlpEndpointUri}", (s) => config.OtlpEndpointUri = s },
```

Comment:
>is going to send traces [](http://example.com/codeflow?start=83&length=23)

I don't think the exporter sends traces to an endpoint, it rather makes them available there, no?